### PR TITLE
announce lcnr as types team co-leadership

### PR DESCRIPTION
**Pull request by nikomatsakis**
r? @jackh726

### CODE REVIEW

File: posts/inside-rust/2024-04-12-types-team-leadership.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: "Announcing lcnr as Types Team co-lead"
+author: Niko Matsakis
+team: the types team <https://www.rust-lang.org/governance/teams/compiler#types-team>
+---
+
+It is my great privilege to announce that [lcnr][] will be serving as the new types team co-lead. The types team has adopted the ["rolling leadership" model](https://rust-lang.github.io/rfcs/3262-compiler-team-rolling-leads.html) pioneered by the compiler team, and so [lcnr][] is joining as the new "junior lead". The senior lead will be [Jack Huey][]. I ([Niko Matsakis][]) am going to be stepping back and I will continue to be active as a types team member.
+
+Over the years [lcnr][] has made numerous contributions to the type system, beginning with a focus on const generics but soon stepping over to the trait solver and type system. His most recent contribution has been leading the push for a new trait solver. This work recently came to its first major milestone, [stabilizing the new solver for use in coherence][#121848]. This was a major goal for the types team and [lcnr][] deserves a lot of credit for it, as well as [compiler-errors][] and [boxyuwu][].
```
Comment (BoxyUwU) [inline posts/inside-rust/2024-04-12-types-team-leadership.md:10]:
```suggestion
Over the years [lcnr][] has made numerous contributions to the type system, beginning with a focus on const generics but soon stepping over to the trait solver and type system. His most recent contribution has been leading the push for a new trait solver. This work recently came to its first major milestone, [stabilizing the new solver for use in coherence][#121848]. This was a major goal for the types team and [lcnr][] deserves a lot of credit for it, as well as [compiler-errors][] and [boxy][].
```

File: posts/inside-rust/2024-04-12-types-team-leadership.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+[#121848]: https://github.com/rust-lang/rust/pull/121848
+[compiler-errors]: https://github.com/compiler-errors
+[boxyuwu]: https://github.com/boxyuwu
```
Comment (BoxyUwU) [inline posts/inside-rust/2024-04-12-types-team-leadership.md:14]:
```suggestion
[boxy]: https://github.com/boxyuwu
```

**Review by BoxyUwU — COMMENTED** [review]
boxyuwu with with no caps feels weird xd

**Comment by jackh726** [comment]
will merge once @BoxyUwU's suggested change is made

**Review by lcnr — APPROVED** [review]

**Pull request merged**